EosPayload/drivers/cutdown_driver.py

```diff
@@ -76,6 +78,12 @@ def cutdown_trigger(self):
 
     @staticmethod
     def cutdown_trigger_mqtt(client, user_data, message):
-        user_data['logger'].info("received cutdown command")
-        user_data['queue'].put(1)
+        try:
+            decoded_msg = CutDown.decode(message)
+            if decoded_msg.get_format_type() != Type.CutDown:
+                raise TypeError("incorrect type")
```

Comment:
don't throw, you should instead use logger.error()
also consider a more descriptive message like "incorrect type X, expected CutDown"